src/main/java/racingcar/Main.java

```diff
@@ -0,0 +1,17 @@
+package racingcar;
+
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        InputView inputView = new InputView(new Scanner(System.in));
+        int carCount = inputView.inputCarCount();
+        int tryCount = inputView.inputTryCount();
```

Comment:
```suggestion
        int carCount = inputView.inputCarCount();
        int tryCount = inputView.inputTryCount();
```

우리 도메인에서 
```bash
자동차 대수는 몇 대 인가요?
-1
시도할 회수는 몇 회 인가요?
-1
```

이러한 입력이 있어도 정상이어야할까요? 유저에게 알려줘야할까요?
해당책임을 담당하는 객체가 있다면 어떨까요?